pydantic_core/core_schema.py

```diff
@@ -3802,6 +3802,7 @@ def definition_reference_schema(
     'int_from_float',
     'float_type',
     'float_parsing',
+    'float_parsing_size',
```

Comment:
do we have a test which actually raises this error?

If it's not possible to actually made it happen, or very rare, I think we just use `float_parsing`.

Reply:
I didn't manage to test it. I replaced it with `float_parsing`